exercises/java/encryption-context-complete/Makefile

```diff
@@ -0,0 +1,6 @@
+coverage:
+	mvn clean
+	mvn test
+	cd ./target/site/jacoco/; python -m http.server &
```

Comment:
can you say python3 here? the syntax for http server changed between 2 and 3

https://www.poftut.com/how-to-run-and-use-simple-http-server-in-python2-and-python3/

Reply:
Yes good point. I had envisioned this going away but maybe it's useful enough to hang around.